akka-docs/rst/java/code/docs/testkit/TestKitDocTest.java

```diff
@@ -207,10 +189,10 @@ public void demonstrateExpect() {
       getRef().tell(42, ActorRef.noSender());
       //#test-expect
       final String hello = expectMsgEquals("hello");
-      final Object   any = expectMsgAnyOf("hello", "world");
-      final Object[] all = expectMsgAllOf("hello", "world");
+      final Object   any = expectMsgAnyOf(Arrays.asList("hello", "world"));
+      final Object[] all = (Object[]) expectMsgAllOf(Arrays.asList("hello", "world"));
```

Comment:
This smells bad, my spidey senses are tingling: should this not really return a `String[]`?